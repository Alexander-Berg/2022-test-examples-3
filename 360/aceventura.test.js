'use strict';

const {
    normalizeContact,
    normalizePopularContact
} = require('./aceventura.js');

test('normalizeContact', () => {
    expect(normalizeContact({ cid: 'cid', email: 'e@ma.il', name: 'name' })).toMatchSnapshot();
});

test('normalizePopularContact', () => {
    expect(normalizePopularContact({ cid: 'cid', email: 'e@ma.il', name: 'name' })).toMatchSnapshot();
});
