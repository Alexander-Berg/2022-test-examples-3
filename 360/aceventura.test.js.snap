// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`normalizeContact 1`] = `
Object {
  "cid": "cid",
  "email": "e@ma.il",
  "name": "name",
}
`;

exports[`normalizePopularContact 1`] = `
Object {
  "cid": "cid",
  "email": "e@ma.il",
  "last_usage": 0,
  "name": Object {
    "first": "name",
    "full": "name",
    "last": "",
  },
}
`;
